WordPress/Classes/ViewRelated/Tools/Time Zone/Views/TimeZoneTableViewCell.swift

```diff
@@ -0,0 +1,132 @@
+import UIKit
+
+class TimeZoneTableViewCell: WPTableViewCell {
+
+    let titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 1
+        let fontDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: .body)
+        label.font = UIFont(descriptor: fontDescriptor, size: fontDescriptor.pointSize)
+        label.adjustsFontForContentSizeCategory = true
+        label.textColor = .label
+
+        if label.effectiveUserInterfaceLayoutDirection == .leftToRight {
+            // swiftlint:disable:next natural_text_alignment
+            label.textAlignment = .left
+        } else {
+            // swiftlint:disable:next inverse_text_alignment
+            label.textAlignment = .right
+        }
+        return label
+    }()
+
+    let leftSubtitle: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 1
+        let fontDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: .caption1)
+        label.font = UIFont(descriptor: fontDescriptor, size: fontDescriptor.pointSize)
+        label.adjustsFontForContentSizeCategory = true
+        label.textColor = .secondaryLabel
+
+        if label.effectiveUserInterfaceLayoutDirection == .leftToRight {
+            // swiftlint:disable:next natural_text_alignment
+            label.textAlignment = .left
+        } else {
+            // swiftlint:disable:next inverse_text_alignment
+            label.textAlignment = .right
+        }
+
+        return label
+    }()
+
+    let rightSubtitle: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 1
+        let fontDescriptor = UIFontDescriptor.preferredFontDescriptor(withTextStyle: .caption1)
+        label.font = UIFont(descriptor: fontDescriptor, size: fontDescriptor.pointSize)
+        label.adjustsFontForContentSizeCategory = true
+        label.textColor = .secondaryLabel
+
+        if label.effectiveUserInterfaceLayoutDirection == .leftToRight {
+            // swiftlint:disable:next inverse_text_alignment
+            label.textAlignment = .right
+        } else {
+            // swiftlint:disable:next natural_text_alignment
+            label.textAlignment = .left
+        }
+
+        return label
+    }()
```

Comment:
These look good!! One more optional comment here: you don't have to do it, but you could also `lazy var`. This could also help with some code optimization calling a method in there